RR.UnitOfWork/Repositories/Shared/EmailHistoryRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using RR.UnitOfWork.Entities;
+using RR.UnitOfWork.Entities.Shared;
+
+namespace RR.UnitOfWork.Repositories.Shared;
+
+public interface IEmailHistoryRepository : IRepository<EmailHistory>
+{
+}
+
+public class EmailHistoryRepository : BaseRepository<EmailHistory>, IEmailHistoryRepository
+{
+    public EmailHistoryRepository(DatabaseContext db) : base(db)
+    { }
+}
```

Comment:
Refactor:  when it comes to brackets, are we going to be flexible in  using both?
{
}
and
{ }

Reply:
We can keep it line separated brackets